### DISCUSSION TITLE
Determinant less than zero.

### DISCUSSION THREAD

**Discussion by kuailebozai** [Anura3D/Anura3D_OpenSource#152] [Q&A]
Hello, when I was using anura3d to calculate soil compression settlement, I encountered "[ERROR]: (time step: 1) Determinant less than zero. [subroutine FormB3()]." I don't know why. Please help me. Thank you very much.
![屏幕截图 2023-11-11 210044](https://github.com/Anura3D/Anura3D_OpenSource/assets/52518762/176efb1a-3b06-444f-bbcc-136d2a87e7cc)
![屏幕截图 2023-11-11 210121](https://github.com/Anura3D/Anura3D_OpenSource/assets/52518762/75eb23da-325b-4d43-9e12-65143df10ebc)

**Comment by icy1237** [comment]
Hello, have you resolved this problem?
